src/lib/dialog/dialog.ts

```diff
@@ -120,6 +120,10 @@ export class MdDialog {
         .centerHorizontally()
         .centerVertically();
 
+    if (dialogConfig.paneClassName) {
```

Comment:
Same thing for dialog really, I could have a fixed element that will have z-index higher than 1000 and it would be overlaying the overlay.